mkdocs/docs/contributing/guide.md

```diff
@@ -0,0 +1,182 @@
+---
+template: main.html
+---
+
+# Contributing Guide
+
+Welcome! We are delighted that you want to contribute to Iter8! 💖
+
+As you get started, you are in the best position to give us feedback on areas of
+our project that we need help with including:
+
+* Problems found during setup of Iter8
+* Gaps in our quick start guide or other tutorials and documentation
+* Bugs in our test and automation scripts
+
+If anything doesn't make sense, or doesn't work when you run it, please open a
+bug report and let us know!
+
+***
+
+## Ways to Contribute
+
+We welcome many different types of contributions including:
+
+* [Iter8 documentation including tutorials](../tutorials)
+* New features
+* Builds, CI
+* Bug fixes
+* Web design for https://iter8.tools
+* Communications / social media / blog posts
+* Reviewing pull requests
+
+Not everything happens through a GitHub pull request. Please come to our
+[meetings](../../../getting-started/help) or [contact us](../../../getting-started/help) and let's discuss how we can work together. 
```

Comment:
I have a nit. link to meetings makes sense but I expected to go to #come-to-meetings because I can see it immediately below. Perhaps it would be a better option?